### PR TITLE
#11179 add more metadata info about packages for Oracle; use constants

### LINKED ISSUE #11179: Creation date / modification date of tables, views and packages
Hallo,
can I see the creation / modification date of tables, views and packages somewhere? Best directly with the properties of the table or package, ... . Unfortunately, I haven't found it anywhere. 
Best regards.

### PR DESCRIPTION
**Pull request by LonwoLonwo**
2023-06-14 18_29_59-Window

https://docs.oracle.com/cd/B14117_01/server.101/b10755/statviews_1102.htm

### CODE REVIEW

File: plugins/org.jkiss.dbeaver.ext.oracle/OSGI-INF/l10n/bundle.properties [hunk 1]
```diff
@@ -181,6 +181,12 @@ meta.org.jkiss.dbeaver.ext.oracle.model.OracleObject.name.name=Name
 meta.org.jkiss.dbeaver.ext.oracle.model.OraclePackage.valid.name=Valid
 meta.org.jkiss.dbeaver.ext.oracle.model.OraclePackage.objectDefinitionText.name=Header
 meta.org.jkiss.dbeaver.ext.oracle.model.OraclePackage.extendedDefinitionText.name=Body
+meta.org.jkiss.dbeaver.ext.oracle.model.OraclePackage.created.name=Created
+meta.org.jkiss.dbeaver.ext.oracle.model.OraclePackage.created.description=Timestamp for the creation of the object
+meta.org.jkiss.dbeaver.ext.oracle.model.OraclePackage.temporary.name=Temporary
+meta.org.jkiss.dbeaver.ext.oracle.model.OraclePackage.temporary.description=Whether the object is temporary (the current session can see only data that it placed in this object itself)
```
Comment (E1izabeth) [inline plugins/org.jkiss.dbeaver.ext.oracle/OSGI-INF/l10n/bundle.properties:187]: Do you mean this?
```suggestion
meta.org.jkiss.dbeaver.ext.oracle.model.OraclePackage.temporary.description=Whether the object is temporary (the current session can see only data that is placed in this object itself)
```

File: plugins/org.jkiss.dbeaver.ext.oracle/OSGI-INF/l10n/bundle.properties [hunk 2]
```diff
@@ -181,6 +181,12 @@ meta.org.jkiss.dbeaver.ext.oracle.model.OracleObject.name.name=Name
...
+meta.org.jkiss.dbeaver.ext.oracle.model.OraclePackage.lastDDLTime.name=Last DDL Time
+meta.org.jkiss.dbeaver.ext.oracle.model.OraclePackage.lastDDLTime.description=Timestamp for the last modification of the object resulting from a DDL statement (including grants and revokes)
```
Comment (E1izabeth) [inline plugins/org.jkiss.dbeaver.ext.oracle/OSGI-INF/l10n/bundle.properties:189]: I don't like `resulting`. For me, it makes sentence unclear. Maybe `due to` or `as a result` fits better.

**Review by E1izabeth — CHANGES_REQUESTED** [review]
Please, fix typo and message formulation..

**Comment by uslss** [comment]
verified

**Pull request merged**